boards/arduino-mkrzero/include/board.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_arduino-mkrzero Arduino MKRZERO
+ * @ingroup     boards
+ * @brief       Support for the Arduino MKRZERO board.
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the Arduino MKRZERO
+ *              board
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "periph_conf.h"
+#include "board_common.h"
+#include "arduino_pinmap.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    SD Card device configuration
+ * @{
+ */
+#define SDCARD_SPI_PARAM_SPI         (SPI_DEV(1))
```

Comment:
It should be ok, regarding the [specs](https://www.arduino.cc/en/uploads/Main/ArduinoMKRZero-schematic.pdf). I just don't know what CD1 is on the SD card reader.